src/Parser/Rogue/Subtlety/Modules/RogueCore/ComboPointDetails.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Tab from 'Main/Tab';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+import SpellLink from 'common/SpellLink';
+
+import ComboPointTracker from './ComboPointTracker';
+
+import ResourceBreakdown from '../ResourceTracker/ResourceBreakdown';
+import resourceSuggest from './../ResourceTracker/ResourceSuggest';
+
+class ComboPointDetails extends Analyzer {
+  static dependencies = {
+    comboPointTracker: ComboPointTracker,
+  };
+  
+
+  makeExtraSuggestion(spell) {
+    return <span>Avoid waisting combo points when casting <SpellLink id={spell.id}  /> </span>
```

Comment:
It's preferred to use the Wrapper component from `common/Wrapper` instead of `span` when you want to return multiple children without a visible container.